README.md

```diff
@@ -134,21 +134,11 @@ With [nuxt](https://nuxtjs.org/) - [nuxt.config.js](https://nuxtjs.org/faq/exten
 // nuxt
 
 module.exports = {
-	build: {
-		extend(config) {
-			const vueRule = config.module.rules.find(({ test }) => test.toString() === /\.vue$/i.toString());
-			vueRule.use = [
-				{
-					loader: vueRule.loader,
-					options: vueRule.options
-				}, {
-					loader: "vue-svg-inline-loader",
-					options: { /* ... */ }
-				}
-			];
-			delete vueRule.loader;
-			delete vueRule.options;
-		}
+	buildModules: ['vue-svg-inline-loader/nuxt', { /* options */ }],
+	// or
+	buildModules: ['vue-svg-inline-loader/nuxt'],
+	svgInlineLoader: {
```

Comment:
This key might be confusing for users - `svgInlineLoader` vs. `vueSvgInlineLoader` vs. `svgInlineLoaderOptions` vs. `vueSvgInlineLoaderOptions` (this should be correct key IMHO).

Reply:
Agreed, but I wouldn't add the "option" suffix. 
Most Nuxt module only use the module name as the option key so I'd go with `vueSvgInlineLoader`

Reply:
`vueSvgInlineLoader` key is fine by me. Feel free to update readme and code.